crates/sui-sdk/src/apis.rs

```diff
@@ -364,6 +787,31 @@ impl CoinReadApi {
         )
     }
 
+    /// Return a list of coins for the given address, or an error upon failure.
+    ///
+    /// Note that the function selects coins to meet or exceed the requested `amount`.
+    /// If that it is not possible, it will fail with an insufficient fund error.
+    ///
+    /// The coins can be filtered by `coin_type` or use `None` for including all coin types.
```

Comment:
The default is actually `Coin<SUI>`, rather than "all coin types" (same applies to the functions this relies on `get_coins_stream` and `get_coins`:

https://github.com/MystenLabs/sui/blob/1d3c74e46a73e0958356ea3ac33883578111c617/crates/sui-json-rpc/src/api/coin.rs#L14-L26

Reply:
Thanks for noticing!